### PR TITLE
Fix problem observed in running on 7 or more GPUs.

### PR DESCRIPTION
**Pull request by ronlieb**
Short term fix, use visited logic to only call one.

### CODE REVIEW

File: openmp/libomptarget/plugins/hsa/src/rtl.cpp [hunk 1]
```diff
@@ -593,10 +593,18 @@ __tgt_target_table *__tgt_rtl_load_binary(int32_t device_id,
     elf_end(elfP);
   }
 
+  atmi_status_t err;
+  // Temporaryily adding a fix to 7 and 8 gpu cards working.
+  // This patch needs to be reverted once we have a fix as described in:
+  // https://github.com/RadeonOpenCompute/atmi-staging/issues/67
```
Comment (ashwinma) [inline openmp/libomptarget/plugins/hsa/src/rtl.cpp]: These are not public links unfortunately. Can we summarize the fix in two bullet points here instead?

File: openmp/libomptarget/plugins/hsa/src/rtl.cpp [hunk 2]
```diff
@@ -593,10 +593,18 @@ __tgt_target_table *__tgt_rtl_load_binary(int32_t device_id,
     elf_end(elfP);
   }
 
+  atmi_status_t err;
+  // Temporaryily adding a fix to 7 and 8 gpu cards working.
```
Comment (ashwinma) [inline openmp/libomptarget/plugins/hsa/src/rtl.cpp]: minor spell check

File: openmp/libomptarget/plugins/hsa/src/rtl.cpp (hunk 1 above)
Comment (ronlieb) [reply openmp/libomptarget/plugins/hsa/src/rtl.cpp]: done

File: openmp/libomptarget/plugins/hsa/src/rtl.cpp (hunk 2 above)
Comment (ronlieb) [reply openmp/libomptarget/plugins/hsa/src/rtl.cpp]: done

**Comment by ashwinma** [comment]
LGTM I suppose

**Pull request merged**